tlslite/constants.py

```diff
@@ -365,6 +367,20 @@ class CertificateStatusType(TLSEnum):
     ocsp = 1
 
 
+class HeartbeatMode(TLSEnum):
+    """Types of heartbeat modes from RFC 6520"""
+
+    PEER_ALLOWED_TO_SEND = 1
+    PEER_NOT_ALLOWED_TO_SEND = 2
+
+
+class HeartbeatMessageType(TLSEnum):
+    """Types of heartbeat messages from RFC 6520"""
+
+    HEARTBEAT_REQUEST = 1
+    HEARTBEAT_RESPONSE = 2
```

Comment:
same here, for consistency and to reflect naming from RFCs, the names should be lower case